admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UserDetailsServiceImpl.java

```diff
@@ -347,6 +347,7 @@ public UserDetailsPutDto updateUser(UserDetailsPutDto userDetailsDto) {
 			UserDetailsHistory udh = new UserDetailsHistory();
 			MapperUtils.map(ud, udh);
 			MapperUtils.setBaseFieldValue(ud, udh);
+			udh.setIsActive(true);

```

Comment:
you shouldnt hardcode it to true @dhanendra06 , it should be based on the row for which we are creating the history

Reply:
taken care